library/std/src/thread/mod.rs

```diff
@@ -1211,7 +1207,7 @@ impl ThreadId {
         }
     }
 
-    #[cfg(not(target_thread_local))]
+    #[allow(unused)]
```

Comment:
Because this is also used when 64-bit atomics are unavailable.

Reply:
Is there any more specific `cfg` that can be used? `#[allow(unused)]`, while sometimes necessary, makes me nervous because it could silently become actual dead code in the future.